packages/components/src/styles/components/tag.scss

```diff
@@ -37,20 +36,32 @@ $hds-tag-border-radius: 50px;
 .hds-tag__dismiss-icon {
   width: 12px;
   height: 12px;
+}
+
+.hds-tag__dismiss-icon,
+.hds-tag__text {
   color: var(--token-color-foreground-primary);
 }
 
+.hds-tag__dismiss,
+.hds-tag__text,
+.hds-tag__link {
+  background-color: var(--token-color-surface-interactive);
```

Comment:
[todo] this has already been applied at line 57, so you can move the declaration under the `.hds-tag__dismiss` and remove this whole block.